components/mod-event/useModEventList.tsx

```diff
@@ -225,6 +222,42 @@ export const useModEventList = (
         queryParams.addedTags = removedTags.trim().split(',')
       }
 
+      const filterTypes = types.filter(Boolean)
+      if (filterTypes.length < allTypes.length && filterTypes.length > 0) {
+        queryParams.types = filterTypes.map((type) => {
+          // There is a no appeal type, it's a placeholder and behind the scene
+          // we use type as report and reportTypes as appeal
+          if (type === MOD_EVENTS.APPEAL) {
+            if (!queryParams.reportTypes) {
+              queryParams.reportTypes = []
+            }
```

Comment:
Nit: I personnaly like using the following notation.

```suggestion
            queryParams.reportTypes ||= []
```